e2e/test/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -1051,6 +1052,31 @@ describe("scenarios > visualizations > pivot tables", { tags: "@slow" }, () => {
       });
     });
   });
+
+  it("should not have to wait for data to show fields in summarisation (metabase#26467)", () => {
+    cy.intercept("POST", "api/card/pivot/*/query", req => {
+      req.on("response", res => {
+        res.setDelay(20_000);
+      });
+    });
+
+    createTestQuestion({ visitQuestion: false }).then(({ body }) => {
+      // manually visiting the question to avoid the auto wait logic,
+      // we need to go to the editor while the query is still loading
+      cy.visit(`/question/${body.id}`);
+    });
+
+    // confirm that it's loading
+    main().findByText("Doing science...").should("be.visible");
```

Comment:
Since now we're not waiting for the query to load, should we at least wait for query metadata to load, to reduce chance of this test flaking?
Note: I think that endpoint gets called more than once.

Reply:
Metadata endpoints seems to be pretty fast, if they run so slow to make cypress timeout I suspect the test run is probably broken anyway, what do you think?

Reply:
Unfortunately GitHub actions machines can be quite slow.
If I'm unsure I check e2e flakiness by using network throttling (Fast 3G) + CPU throttling (4x-6x) locally before submitting them.

> Metadata endpoints seems to be pretty fast

Fair point :+1: